### PR TITLE
Add chinese missing link for download page

### PR DESCRIPTION
**Pull request by wongzigii**
Fix https://github.com/nodejs/nodejs.org/issues/846

### CODE REVIEW

**Comment by lpinca** [comment]
LGTM if the value of the `text` field is correct :)

**Comment by phillipj** [comment]
Hm, this shows we've got a bug in the site.json fallback logic found in [build.js#L44](https://github.com/nodejs/nodejs.org/blob/master/build.js#L44), right?

..ofcourse not a blocker for this PR.

**Comment by lpinca** [comment]
@phillipj correct, `Object.assign` is not suitable in this case. We need to use https://lodash.com/docs#defaultsDeep or something like that.

**Comment by lpinca** [comment]
Looks like #848 is not enough to fix the fallback issue because the page uses [`site.download.shasums.text`](https://github.com/nodejs/nodejs.org/blob/024130aa6805dda4a44e23576be0fb318373d6f9/layouts/partials/download-list.hbs#L3) and [`site`](https://github.com/nodejs/nodejs.org/blob/024130aa6805dda4a44e23576be0fb318373d6f9/build.js#L65) is actually the parsed contents of the `site.json` not the result of the [`i18nJSON`](https://github.com/nodejs/nodejs.org/blob/024130aa6805dda4a44e23576be0fb318373d6f9/build.js#L48-L53) function.

This fallback thing needs a little work. This is what I have in mind:
- Remove `i18n` [here](https://github.com/nodejs/nodejs.org/blob/024130aa6805dda4a44e23576be0fb318373d6f9/build.js#L67) and the [i18n helper](https://github.com/nodejs/nodejs.org/blob/024130aa6805dda4a44e23576be0fb318373d6f9/scripts/helpers/i18n.js).
- Make `site` be the returned value of the `i18nJSON` function.
- Use `site` everywhere.

**Comment by lpinca** [comment]
cc: @nodejs/nodejs-cn

File: locale/zh-cn/site.json [hunk 1]
```diff
@@ -71,6 +71,10 @@
         "package-manager": {
             "link": "download/package-manager",
             "text": "使用包管理器安装 Node.js"
+        },
+        "shasums": {
+          "link": "SHASUMS256.txt.asc",
+          "text": "查看已签名的 SHASUM"
         }
```
Comment (fundon) [inline locale/zh-cn/site.json:78]: “查看发布文件的 SHASUM 签名“ 不知道如何？

**Comment by fengmk2** [comment]
+1

**Comment by laosb** [comment]
LGTM except @fundon 's comment above.

File: locale/zh-cn/site.json (hunk 1 above)
Comment (wongzigii) [inline locale/zh-cn/site.json]: 查看已发布版本的 SHASUM 签名？

**Comment by wongzigii** [comment]
Just update `site.json` for a better translation! Ready for merge. 😄

**Comment by wongzigii** [comment]
Thanks for reviewing!

**Comment by lpinca** [comment]
Thank you @wongzigii.

**Pull request merged**